packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/dataStreamSearch/dataStreamSearch.tsx

```diff
@@ -20,12 +18,28 @@ export function DataStreamSearch({ onSubmit, client }: DataStreamSearchProps) {
     defaultValues: { workspace: null, searchQuery: '' },
   });
 
+  const [isError, setIsError] = useState<boolean>();
+
   return (
     <SpaceBetween size='xxs'>
       <Header variant='h3' description='Search for modeled data streams by name or by the associated asset.'>
         Search modeled data streams
       </Header>
 
+      {isError && (
```

Comment:
proabably worth moving this block to a different file / component

Reply:
resolved